docs/src/man/convex_sets.md

```diff
@@ -0,0 +1,258 @@
+# Introduction to Convex Sets
+
+In this section we give a short overview about convex sets supported in this
+library and the corresponding lazy set calculus using support functions.
+
+```@contents
+Pages = ["convex_sets.md"]
+Depth = 4
+```
+
+```@setup convex_sets
+using LazySets, Plots, Plots.PlotMeasures, LaTeXStrings
+
+c1 = "blue"
+c2 = "green"
+c3 = "red"
+c4 = "orange"
+
+function plot_raw()
+    plot(tickfont=font(30, "Times"), guidefontsize=45, aspectratio=1,
+         size=(300, 300), leg=false)
+end
+
+function plotX(X, ε, modify, color; linecolor="black")
+    if !modify
+        plot_raw()
+    end
+    plot!(X, ε, color=color, alpha=0.7, linecolor=linecolor)
+end
+
+function plotX(X, modify, color; linecolor="black")
+    if !modify
+        plot_raw()
+    end
+    plot!(X, color=color, alpha=0.7, linecolor=linecolor)
+end
+
+function plot1(X, modify::Bool=false; linecolor="black")
+    plotX(X, modify, c1, linecolor=linecolor)
+end
+
+function plot2(X, modify::Bool=false; linecolor="black")
+    plotX(X, modify, c2, linecolor=linecolor)
+end
+
+function plot3(X, modify::Bool=false; linecolor="black")
+    plotX(X, modify, c3, linecolor=linecolor)
+end
+
+function plot1(X, ε::Float64, modify::Bool=false; linecolor="black")
+    plotX(X, ε, modify, c1, linecolor=linecolor)
+end
+
+function plot2(X, ε::Float64, modify::Bool=false; linecolor="black")
+    plotX(X, ε, modify, c2, linecolor=linecolor)
+end
+
+function plot3(X, ε::Float64, modify::Bool=false; linecolor="black")
+    plotX(X, ε, modify, c3, linecolor=linecolor)
+end
+
+function plot_hyperplane!(x, y, linecolor)
+    plot!(x, y, linecolor=linecolor, arrow=:arrow, width=8)
+    plot!([x[2], x[1]], [y[2], y[1]], linecolor=linecolor, arrow=:arrow, width=8)
+end
+```
+
+## Basic convex sets
+
+A set ``X ⊆ ℝⁿ`` is convex if for any two points ``x``, ``y`` in ``X`` the whole
```

Comment:
We discussed this elsewhere, i know, but reading the ``ℝⁿ`` symbol in the generated doc looks like something is wrong with it the symbol. Since this is not docstring, we can use ``\\mathbb{R}^n`` which properly aligned the super-index ``n``.